src/defaultsettings.cpp

```diff
@@ -324,14 +324,17 @@ void set_default_settings()
 	settings->setDefault("enable_waving_plants", "false");
 	settings->setDefault("exposure_compensation", "0.0");
 	settings->setDefault("enable_auto_exposure", "false");
+	settings->setDefault("secondstage_gamma", "1.6");
 	settings->setDefault("debanding", "true");
 	settings->setDefault("antialiasing", "none");
 	settings->setDefault("fxaa", "false");
 	settings->setDefault("enable_bloom", "false");
 	settings->setDefault("enable_bloom_debug", "false");
 	settings->setDefault("enable_volumetric_lighting", "false");
+	settings->setDefault("enable_volumetric_depth_attenuation", "false");
 	settings->setDefault("enable_water_reflections", "false");
 	settings->setDefault("enable_translucent_foliage", "false");
+	settings->setDefault("enable_tinted_fog", "false");
```

Comment:
this was enabled before. why the change?

Reply:
Most effects are disabled by default, so it seemed natural that the same should be true here.

Reply:
but this is not a new effect. it should be enabled by default to match the previous behavior.

Reply:
I suppose that makes sense.